src/assets/scss/app.scss

```diff
@@ -1154,6 +1154,28 @@ html {
     --bs-table-active-color: var(--bs-body-color);
     --bs-table-hover-color: var(--bs-body-color);
   }
+
+  /****** classes referenced by webapps ******/
+  .chaise-content-left {
```

Comment:
This is something that we should discuss in the meeting, but do we want these classes to be only available under markdown content? are we planning to use these in the internal implementation? or maybe we want to encourage static sites to use these?